Sandbox/Makefile

```diff
@@ -107,6 +118,7 @@ image_java: \
 
 push: image image_java
 	$(call push_image,microfn/sandbox)
+	$(call push_image,microfn/sandbox_gpu)
```

Comment:
microfn/sandbox_java_gpu?

Need to also update the dependencies for the Makefile target.